docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -22,6 +22,7 @@ Excel add-ins run across multiple versions of Office, including Office 2016 or l
 |  Requirement set  |  Office on Windows<br>(connected to Office 365 subscription)  |  Office on iPad<br>(connected to Office 365 subscription)  |  Office on Mac<br>(connected to Office 365 subscription)  | Office on the web |
 |:-----|-----|:-----|:-----|:-----|:-----|
 | [Preview](excel-preview-apis.md)  | Please use the latest Office version to try preview APIs (you may need to join the [Office Insider program](https://products.office.com/office-insider)) |
+| [ExcelApi 1.10](excel-api-1-10-requirement-set.md) | Version 1907 (Build 11929.20306) or later | 2.30 or later | 16.30 or later | October 2019 |
```

Comment:
The page https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-1-10-requirement-set?branch=AlexJ-NewExcelMonikers exists, but isn't listed in the TOC. Should add a link for it in the TOC.